src/torchmetrics/classification/confusion_matrix.py

```diff
@@ -199,7 +199,7 @@ class MulticlassConfusionMatrix(Metric):
 
     - :math:`C_{i, i}` represents the number of true positives for class :math:`i`
     - :math:`\sum_{j=1, j\neq i}^N C_{i, j}` represents the number of false negatives for class :math:`i`
-    - :math:`\sum_{i=1, i\neq j}^N C_{i, j}` represents the number of false positives for class :math:`i`
+    - :math:`\sum_{i=1, i\neq j}^N C_{i, j}` represents the number of false positives for class :math:`j`
     - the sum of the remaining cells in the matrix represents the number of true negatives for class :math:`i`
```

Comment:
I noticed that this sentence refers to `i` again, which does not make much sense anymore if we refer to `j` in L202.
I will change it back to `i`, and swap `i` and `j` in the equation.

If swapping is not desired, the "target class" in the equations could be switched to something else, e.g. `k`.

Reply:
Updated.

Reply:
cc: @SkafteNicki